superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -163,7 +163,8 @@ export default class FilterableTable extends PureComponent<
   }
 
   componentDidMount() {
-    this.fitTableToWidthIfNeeded();
+    if (this.container.current)
```

Comment:
since this is already on 2 lines, maybe we should add the braces around it anyway?

I've been meaning to look into why this isn't enforced by our lint setup currently: https://github.com/airbnb/javascript#blocks--braces